pkgs/top-level/all-packages.nix

```
@@ -1872,6 +1872,8 @@ in
 
   optar = callPackage ../tools/graphics/optar {};
 
+  pastel = callPackage ../applications/misc/pastel { };
```

I'll throw you a bone to fix the eval, :)
```suggestion
  pastel = callPackage ../applications/misc/pastel {
    inherit (darwin.apple_sdk.frameworks) Security;
  };
```

---

Thanks! I had done this in my local configuration but it completely slipped my mind when making this PR.